scripts/tools/quad_dual_pad_border.py

```diff
@@ -100,7 +114,8 @@ def add_quad_pad_border(kicad_mod, pad_details, device_params, pad_shape_details
             chamfer_corner_selection_first=corner_first,
             chamfer_corner_selection_last=corner_last,
             end_pads_size_reduction = pad_size_reduction,
-            **pad_details['left'], **pad_shape_details)
+            **pad_details['left'], **pad_shape_details,
+            increment=increment)
```

Comment:
I would suggest moving the increment stuff before the unpacked dicts for consistency

Reply:
Done